README.md

```diff
@@ -9,16 +9,32 @@ Bowling Challenge
 
 ## The Task
 
-**THIS IS NOT A BOWLING GAME, IT IS A BOWLING SCORECARD. DO NOT GENERATE RANDOM ROLLS. THE USER INPUTS THE ROLLS.**
+**THIS IS NOT A BOWLING GAME, IT IS A BOWLING SCORECARD. DO NOT GENERATE RANDOM ROLLS. AN ACTUAL USER INTERFACE IS OPTIONAL**
 
-Count and sum the scores of a bowling game for one player (in JavaScript).
+Count and sum the scores of a bowling game for one player (in JavaScript). 
 
 A bowling game consists of 10 frames in which the player tries to knock down the 10 pins. In every frame the player can roll one or two times. The actual number depends on strikes and spares. The score of a frame is the number of knocked down pins plus bonuses for strikes and spares. After every frame the 10 pins are reset.
 
+Start by looking in detail at the rules and the example of scoring for a complete game given below.
+
+An example of how your code might be used could be:
+
+```javaScript
+let scorecard = new Scorecard()
+scorecard.addRoll(5)
+scorecard.calculateScore() // returns 5
+scorecard.addFrame([3, 5]) // returns "Please finish the current frame [5, ?]"
```

Comment:
Sorry just noticed this — I reckon we should only keep the method `addFrame()` and maybe have it take two arguments (first and second roll), so we remove the complexity of having to check the array size, or verify the current frame is correctly finished, etc. If the learner wants to implement extra methods in their solution, it's up to them, but at least this way we don't derive their attention to the main problem (calculating the score from the rolls data)

```js
let scorecard = new Scorecard()
scorecard.addFrame(5, 0)
scorecard.calculateScore() // returns 5
scorecard.addFrame(3, 5) 
```

Let me know what you think of this :)

Reply:
This makes sense, I've removed addRoll. So they can focus on addFrame and calculateScore, but add methods they think are useful.